Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -790,9 +861,13 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
         /// </summary>
         public void OnInputDown(InputEventData eventData)
         {
-            EndTouch(eventData.SourceId);
-            SetHandDataFromController(eventData.InputSource.Pointers[0].Controller, false);
-            eventData.Use();
+            if (eventData.MixedRealityInputAction.Description != "None")
```

Comment:
Could you explain why you are doing a string compare against "None" here? I am wondering if there's a different way to write this to avoid the string compare...

Reply:
I can find a better test. This check is in there because a trigger pull on  a MC Controller fires several InputDown events- the description is 'None' on all but the last one which is the correct event to capture.